css/style.css

```diff
@@ -0,0 +1,81 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  border: 0;
+}
+
+body {
+  background-color: #e5e5e5;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  padding: 54px 20px;
+  background-color: #fff;
+  position: relative;
+}
+
+.my-logo {
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 19px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.headline {
+  background-color: #fff;
+  background-image: url(/images/mobile-version-bg.png);
```

Comment:
- [ ] It has been a great milestone to use a background position :+1: . However the image is not showing as expected. Make sure the image path is set correctly in the background-image url. Once you have made sure that your CSS file is linked correctly, also check that the image itself is set correctly. Again, you will want to open your code inspector in the browser to check. You may need to refer to this [stackoverflow conversation](https://stackoverflow.com/questions/21374534/css-background-image-not-loading) for clear understanding of image path :relaxed: